great_expectations/render/renderer/site_builder.py

```diff
@@ -673,25 +673,23 @@ def _get_call_to_action_buttons(self, usage_statistics):
         """
         create_expectations = CallToActionButton(
             "How to Create Expectations",
-            # TODO update this link to a proper tutorial
-            "https://docs.greatexpectations.io/en/latest/guides/how_to_guides/creating_and_editing_expectations.html",
+            "https://docs.greatexpectations.io/docs/tutorials/getting_started/tutorial_create_expectations",
```

Comment:
Maybe this instead? I don't feel that strongly about this, though.

https://docs.greatexpectations.io/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data

Reply:
I agree with @talagluck regarding the link to use, since the tutorial pages are meant to be read together while the how to guides can be considered full guides in their own right.